python/cudf/cudf/utils/dtypes.py

```diff
@@ -193,24 +193,76 @@ def is_list_like(obj):
     This function checks if the given `obj`
     is a list-like (list, tuple, Series...)
     type or not.
-
     Parameters
     ----------
     obj : object of any type which needs to be validated.
-
     Returns
     -------
     Boolean: True or False depending on whether the
     input `obj` is like-like or not.
     """
-    return isinstance(obj, (Sequence,),) and not isinstance(obj, (str, bytes))
+    from collections.abc import Sequence
 
     if isinstance(obj, (Sequence, np.ndarray)) and not isinstance(obj, (str, bytes)):
         return True
     else:
         return False
```

Comment:
bad merge? Sequence is imported above, should just be the return line along with adding `np.ndarray` if needed.

Reply:
I think so yes. Fixing